easybuild/easyconfigs/s/stpipeline/stpipeline-1.8.1-foss-2022a.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'stpipeline'
+version = '1.8.1'
+
+homepage = 'https://github.com/SpatialTranscriptomicsResearch/st_pipeline'
+description = """The ST Pipeline contains the tools and scripts needed to process
+ and analyze the raw files generated with the Spatial Transcriptomics method
+ in FASTQ format to generated datasets for down-stream analysis. The ST
+ pipeline can also be used to process single cell data as long as a file
+ with barcodes identifying each cell is provided. The ST Pipeline can also
+ process RNA-Seq datasets generated with or without UMIs."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Pysam', '0.19.1'),
+    ('scikit-learn', '1.1.2'),
+    ('Seaborn', '0.12.1'),
+    ('HTSeq', '2.0.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
```

Comment:
As of EB 5 these are the defaults and should be dropped from the EC.

Reply:
True. I simply did not look into that PR yet. :-)
I have updated it, as requested.